nvflare/private/fed/client/fed_client_base.py

```diff
@@ -147,7 +148,8 @@ def send_aux_message(self, project_name, topic: str, shareable: Shareable, timeo
         :param shareable: Shareable object to submit to server
         """
         try:
-            self.logger.info("Starting to send aux messagee.")
+            # self.logger.info("Starting to send aux messagee.")
```

Comment:
we should remove the commented out lines here as well.

Reply:
Removed.